Extensions/Video/JsExtension.js

```diff
@@ -62,12 +62,6 @@ module.exports = {
     videoObject.getProperties = function (objectContent) {
       var objectProperties = new gd.MapStringPropertyDescriptor();
 
-      objectProperties
```

Comment:
I went a bit full steam ahead on this and deleted the prop altogether.
Should we just deprecate it and still use it in case it's defined? It's complexifying the code and I feel it's not used that much?